src/applications/personalization/profile/components/direct-deposit/DirectDepositV2.jsx

```diff
@@ -0,0 +1,87 @@
+import React from 'react';
+import ReactCSSTransitionGroup from 'react-transition-group/CSSTransitionGroup';
+import AlertBox from '@department-of-veterans-affairs/formation-react/AlertBox';
+
+import DowntimeNotification, {
+  externalServices,
+} from 'platform/monitoring/DowntimeNotification';
+import { focusElement } from 'platform/utilities/ui';
+
+import { handleDowntimeForSection } from '../alerts/DowntimeBanner';
+
+import FraudVictimAlert from './FraudVictimAlert';
+import PaymentHistory from './PaymentHistory';
+import BankInfoCNP from './BankInfoCNPv2';
```

Comment:
Can we name this import `BankInfoCNPv2`?

Reply:
done!